hashicorp/waypoint
# ui: 0.7 CSS changes, details page updates

## Changes
- Removed app cards on application overview screen
- Changed border to bottom of Section component instead of underlining the title
- Changed order of tabs on application overview screen to match new designs
- Added `PanelHeader` component for details page headers (to not interfere with styling of the existing `PageHeader` component)
- Added a `isExpandable` param to the `Section` component to remove collapsibility of a section
- Updated headings on artifact list pages (`h3` -> `h2`) to comply with a11y warnings (styling maintained)

## Related Changes Not Addressed in this PR
- [ ] Remove unused components (PR is long enough and these removals won't affect end user)
  -  Related styling can be removed as well

## Screenshots
### Application Overview Screen Before
<img width="1440" alt="Screen Shot 2021-10-28 at 3 31 51 PM" src="https://user-images.githubusercontent.com/60155296/139323061-b56048e9-16ec-40ac-8abe-1eae0a990a08.png">

### Application Overview Screen After
<img width="1435" alt="Screen Shot 2021-10-28 at 3 26 18 PM" src="https://user-images.githubusercontent.com/60155296/139322355-46b31635-6602-4010-8032-a189d55c533c.png">

### Deployment Details Page Before
<img width="1438" alt="Screen Shot 2021-10-28 at 3 32 25 PM" src="https://user-images.githubusercontent.com/60155296/139323096-901431b4-0fae-4fd4-adb5-24810d15c356.png">

### Deployment Details Page After
<img width="1073" alt="Screen Shot 2021-10-29 at 3 46 42 PM" src="https://user-images.githubusercontent.com/60155296/139493480-6c591566-6aba-43d7-a9ff-bd8484b9fc61.png">

### Build Details Page Before
<img width="1440" alt="Screen Shot 2021-10-28 at 3 33 48 PM" src="https://user-images.githubusercontent.com/60155296/139323323-bdfe60ac-a3c7-4d47-9b83-14a654facdda.png">

### Build Details Page After
<img width="1075" alt="Screen Shot 2021-10-29 at 3 47 42 PM" src="https://user-images.githubusercontent.com/60155296/139493582-4036c137-439d-4b38-9dc3-2351ae5dff8f.png">

### Release Details Page Before
<img width="1440" alt="Screen Shot 2021-10-28 at 3 34 38 PM" src="https://user-images.githubusercontent.com/60155296/139323375-9e529700-9738-44cb-9f5c-f3055ba4e588.png">

### Release Details Page After
<img width="1439" alt="Screen Shot 2021-10-28 at 3 26 44 PM" src="https://user-images.githubusercontent.com/60155296/139322363-81e8405a-ae78-4066-8f69-9a57697e028a.png">

## How to Test
1. Pull down this branch (`git checkout ui/0.7-small-changes`)
2. Start up your dev environment with either:
  3. Mirage data `ember serve` or `yarn start`
  3. Local server (see: [here](https://www.waypointproject.io/docs/server/run)) then `ember serve local`
4. Click around to the application overview page and the details pages for builds, deployments, and releases
